src/styles/blocks/screen.scss

```diff
@@ -0,0 +1,12 @@
+.stop-watch {
```

Comment:
name of file (screen.scss) should be the same with class name of block (.stop-watch)